### DISCUSSION TITLE
Why doesn't it work with multiple images in one image?

### DISCUSSION THREAD

**Discussion by erkinkhidirov** [infinitered/nsfwjs#829] [Questions]
I created a canvas from several images, placed an erotic picture in the middle, and NSFW can't determine whether it's erotic or not. If it's a single image, NSFW works, but if there are multiple different images in one, NSFW doesn't work.

**Comment by GantMan** [comment]
I can elaborate on this, but this is a limitation of discriminative models.  The best way to fix this is to employ a sliding window technique and run the model across another dimension of analytics.
